assets/stylesheets/kitten/components/buttons/_tag-button.scss

```diff
@@ -66,10 +86,34 @@
     border-radius: $tag-button-radius;
     min-height: $tag-button-line-height;
     min-width: $tag-button-min-width;
-  }
 
-  .k-TagButton--hydrogen {
-    @include k-buttonColors('hydrogen', (svg: false));
+    color: $color;
+    background-color: $background-color;
+    border-color: $border-color;
+
+    &:focus {
+      color: $color-focus;
+      background-color: $background-color-focus;
+      border-color: $border-color-focus;
+    }
+
+    &:hover {
+      color: $color-hover;
+      background-color: $background-color-hover;
+      border-color: $border-color-hover;
+    }
+
+    &:active {
```

Comment:
On peut ajouter `.is-selected` ici pour gagner quelques lignes.

Reply:
ah j'ai hésité !! ok je le change